src/main/resources/standardTestCases/globbingBehavior/globbingBehavior.wdl

```diff
@@ -0,0 +1,66 @@
+task writeFiles {
+String dir
+  command <<<
+    mkdir ${dir}
+    echo "hullo" > ${dir}/hello.txt
+    echo "buh-bye" > ${dir}/ciao.txt
+
+    echo "what" > what.txt
+    echo "how" > how.txt
+    echo "why" > why.txt
+
+    sleep 2
+  >>>
+  output {
+    Array[File] strInterpolation_1 = glob(dir + "/*.txt")
+    Array[File] strInterpolation_2=  glob("${dir}/*.txt")
+
+    Array[File] callDirectory = glob("*.txt")
+    Array[File] specificDirectory = glob("out/*.txt")
+  }
+  runtime {
+    docker:"ubuntu:latest"
+  }
+}
+
+task countArrays {
+Array[String] in1
+Array[String] in2
+Array[String] in3
+Array[String] in4
+
+  command <<<
+    python <<CODE
+    arr1 = ["${sep='", "' in1}"]
+    arr2 = ["${sep='", "' in2}"]
+    arr3 = ["${sep='", "' in3}"]
+    arr4 = ["${sep='", "' in4}"]
+
+    print len(arr1)
+    print len(arr2)
+    print len(arr3)
+    print len(arr4)
+
+    CODE
+  >>>
+  output {
+     Array[Int] out = read_lines(stdout())
+  }
+  runtime{
+    docker: "python:2.7"
+  }
+}
+
+workflow globbingBehavior {
+ String outputDir = "out"
+
+ call writeFiles { input: dir = outputDir }
+ call countArrays {input:in1 = writeFiles.strInterpolation_1,
```

Comment:
ToL2: It would be nice if this workflow listed the file contents so we could have a more accurate output validation. Maybe we could do this instead of the `count_arrays` call?
```
scatter (f in writeFiles.strInterpolation_1) {
  String strInterpolation_1_contents = read_string(f)
}
# And for the other arrays ...
# ...
output {
  strInterpolation_1_contents
  # And the others
}
```